packages/fuels/src/cli/utils/fuelsVersionCache.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs';
+import path from 'path';
+
+export const FUELS_VERSION_CACHE_FILE = path.join(__dirname, '.fuels-cache.json');
```

Comment:
Is it worthwhile caching this file external to the project?

Maybe in `os.tmpdir()/.fuels/fuels-cache.json`

Reply:
@Dhaiwat10 Any reason why you did not use `node-cache`, from my [first](https://github.com/FuelLabs/fuels-ts/pull/3108#issuecomment-2346355732) and [second](https://github.com/FuelLabs/fuels-ts/pull/3108#discussion_r1761914663) suggestions?

Reply:
@arboleya primarily because it's in-memory.

Reply:
@Dhaiwat10 Fair point. 🤦

Reply:
If we use a global path, we must consider and prevent multiple installations from writing to the same file.

Reply:
That should never happen because different installations/versions usually have different folders inside the `node_modules` folder

Reply:
Aplogies, I was referring to the `os.tmpdir()` suggestion. cc @petertonysmith94